### PR TITLE
Use odycdn CDN and change image files sizes

### PR DESCRIPTION
**Pull request by kekkyojin**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change

## PR Type

What kind of change does this PR introduce?

- [x] Refactoring (no functional changes)

## Fixes

Issue Number: Partially #232

## Other information
Using sizes extracted from Network tab after making requests to Odysee website

For channel profile pictures: 160 x 160 q = 85
For channel cover picture: 0 x 0, q = 95
For thumbnails pictures: 390 x 220, q = 85

### CODE REVIEW

**Review by akinwale — CHANGES_REQUESTED** [review]
Please declare the numeric values as constants, CHANNEL_THUMBNAIL_WIDTH/HEIGHT, etc. Thanks.

**Review by akinwale — APPROVED** [review]
Looks good.

File: app/src/main/java/com/odysee/app/adapter/ChannelFilterListAdapter.java [hunk 1]
```diff
@@ -95,7 +96,7 @@ public void onBindViewHolder(ChannelFilterListAdapter.ViewHolder vh, int positio
         vh.allView.setVisibility(claim.isPlaceholder() ? View.VISIBLE : View.GONE);
 
         vh.titleView.setText(Helper.isNullOrEmpty(claim.getTitle()) ? claim.getName() : claim.getTitle());
-        String thumbnailUrl = claim.getThumbnailUrl(vh.thumbnailView.getLayoutParams().width, vh.thumbnailView.getLayoutParams().height, 85);
+        String thumbnailUrl = claim.getThumbnailUrl(Utils.CHANNEL_THUMBNAIL_WIDTH, Utils.CHANNEL_THUMBNAIL_HEIGHT, 85);
```
Comment (tzarebczan) [inline app/src/main/java/com/odysee/app/adapter/ChannelFilterListAdapter.java:99]: we can update this one to 95 quality also, recently did that on Odysee.com

File: app/src/main/java/com/odysee/app/adapter/CommentListAdapter.java [hunk 2]
```diff
@@ -479,7 +480,7 @@ public void onClick(View view) {
         int bgColor = Helper.generateRandomColorForValue(comment.getChannelId());
         Helper.setIconViewBackgroundColor(holder.noThumbnailView, bgColor, false, context);
         if (hasThumbnail) {
-            Glide.with(context.getApplicationContext()).asBitmap().load(comment.getPoster().getThumbnailUrl(holder.thumbnailView.getLayoutParams().width, holder.thumbnailView.getLayoutParams().height, 85)).
+            Glide.with(context.getApplicationContext()).asBitmap().load(comment.getPoster().getThumbnailUrl(Utils.CHANNEL_THUMBNAIL_WIDTH, Utils.CHANNEL_THUMBNAIL_HEIGHT, 85)).
```
Comment (tzarebczan) [inline app/src/main/java/com/odysee/app/adapter/CommentListAdapter.java:483]: 95 here too

File: app/src/main/java/com/odysee/app/adapter/NotificationListAdapter.java [hunk 3]
```diff
@@ -205,15 +207,17 @@ public void onBindViewHolder(NotificationListAdapter.ViewHolder vh, int position
...
             String turl;
 
-            if (notification.getCommentAuthor() != null)
-                turl = notification.getCommentAuthor().getThumbnailUrl(vh.authorThumbnailView.getLayoutParams().width, vh.authorThumbnailView.getLayoutParams().height, 85);
-            else {
-                turl = getThumbnailUrl(vh.authorThumbnailView, notification.getAuthorThumbnailUrl());
+            if (notification.getCommentAuthor() != null) {
+                turl = notification.getCommentAuthor().getThumbnailUrl(Utils.CHANNEL_THUMBNAIL_WIDTH, Utils.CHANNEL_THUMBNAIL_HEIGHT, 85);
+            } else {
+                ImageCDNUrl imageCDNUrl = new ImageCDNUrl(Utils.CHANNEL_THUMBNAIL_WIDTH, Utils.CHANNEL_THUMBNAIL_HEIGHT, 85, null, notification.getAuthorThumbnailUrl());
```
Comment (tzarebczan) [inline app/src/main/java/com/odysee/app/adapter/NotificationListAdapter.java:219]: 95 here too

File: app/src/main/java/com/odysee/app/adapter/YouTubeSyncItemAdapter.java [hunk 4]
```diff
@@ -72,7 +72,7 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
 
         String thumbnailUrl = null;
         if (item.getClaim() != null) {
-            thumbnailUrl = item.getClaim().getThumbnailUrl(holder.thumbnailView.getLayoutParams().width, holder.thumbnailView.getLayoutParams().height, 80);
+            thumbnailUrl = item.getClaim().getThumbnailUrl(Utils.STREAM_THUMBNAIL_WIDTH, Utils. STREAM_THUMBNAIL_HEIGHT, 80);
```
Comment (tzarebczan) [inline app/src/main/java/com/odysee/app/adapter/YouTubeSyncItemAdapter.java:75]: should be 85

File: app/src/main/java/com/odysee/app/utils/Utils.java [hunk 5]
```diff
@@ -52,6 +52,13 @@ public final class Utils {
     private static final String SP_ENCRYPTION_KEY = "key";
     private static final String SP_API_SECRET_KEY = "api_secret";
 
+    public static final int CHANNEL_THUMBNAIL_WIDTH = 160;
```
Comment (tzarebczan) [inline app/src/main/java/com/odysee/app/utils/Utils.java:55]: we are using the 200x200 version in the 160 area on Odysee. Can you use that, or 100x100?"

Comment (kekkyojin) [reply app/src/main/java/com/odysee/app/utils/Utils.java]: Using 200x200 for channel profiles shows OK on my low-end tablet, so I think 200x200 is a good value for all devices. Android will resize it if needed.

File: app/src/main/java/com/odysee/app/adapter/YouTubeSyncItemAdapter.java [hunk 6]
```diff
@@ -72,7 +72,7 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
 
         String thumbnailUrl = null;
         if (item.getClaim() != null) {
-            thumbnailUrl = item.getClaim().getThumbnailUrl(Utils.STREAM_THUMBNAIL_WIDTH, Utils. STREAM_THUMBNAIL_HEIGHT, 80);
+            thumbnailUrl = item.getClaim().getThumbnailUrl(Utils.STREAM_THUMBNAIL_WIDTH, Utils. STREAM_THUMBNAIL_HEIGHT, 85);
```
Comment (tzarebczan) [inline app/src/main/java/com/odysee/app/adapter/YouTubeSyncItemAdapter.java:75]: Will fix these up when I do a review

**Comment by tzarebczan** [comment]
@akinwale updated these, and caught a few other instances. Can you give it a quick sanity check?

**Review by akinwale — APPROVED** [review]
Everything looks good to me. @kekkyojin You can merge when you're ready.

**Comment by kekkyojin** [comment]
Hmmmm this doesn’t seem to be my original PR, but merging anyway

**Pull request merged**